source/common/http/codec_client.h

```diff
@@ -63,6 +63,12 @@ class CodecClient : Logger::Loggable<Logger::Id::client>,
     connection_->addConnectionCallbacks(cb);
   }
 
+  /**
+   * Return flag indicating whether half-close semantics are enabled on the
```

Comment:
Let's avoid "flag" here

Return if half-close semantics are enabled on the underlying connection
?